client/checkout/platform-checkout/email-input-iframe.js

```diff
@@ -308,9 +325,11 @@ export const handlePlatformCheckoutEmailInput = async ( field, api ) => {
 					spinner.remove();
 					return;
 				}
-			} catch {
-				showErrorMessage();
-				spinner.remove();
+			} catch ( err ) {
+				if ( 20 !== err.code ) {
```

Comment:
Here, and in the later usage, could we instead check for whether `err.name` is `AbortError` instead of checking for the numeric code 20? For future readability I think it may be a little unclear what `20` means in this context.

Reply:
Fixed on https://github.com/Automattic/woocommerce-payments/pull/4803/commits/5ad704780f95561ca8beac7ceb19874636a48885